frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -618,14 +625,28 @@ public static function apiResolve(\OmegaUp\Request $r): array {
                 $problemParams->updatePublished,
                 /*$redirect=*/ false
             );
-            \OmegaUp\DAO\QualityNominations::update($qualitynomination);
-            \OmegaUp\DAO\QualityNominationLog::create($qualitynominationlog);
+            foreach ($nominations as $nomination) {
+                $nomination->status = strval($r['status']);
+                \OmegaUp\DAO\QualityNominations::update($nomination);
+                \OmegaUp\DAO\QualityNominationLog::create(
+                    new \OmegaUp\DAO\VO\QualityNominationLog([
+                        'user_id' => $nomination->user_id,
+                        'qualitynomination_id' => $nomination->qualitynomination_id,
+                        'from_status' => $nomination->status,
+                        'to_status' => $r['status'],
+                        'rationale' => $r['rationale']
+                    ])
+                );
+            }
             \OmegaUp\DAO\DAO::transEnd();
-            if ($r['status'] == 'banned' || $r['status'] == 'warning') {
+            if (
+                $r['status'] == 'banned'
+                || $r['status'] == 'warning'
+            ) {
```

Comment:
nit:

```suggestion
            if ($r['status'] == 'banned' || $r['status'] == 'warning') {
```

esto se puede quedar como estaba.

Reply:
listo